civictechprojects/views.py

```diff
@@ -338,10 +338,12 @@ def volunteer_with_project(request, project_id):
 @csrf_exempt
 def accept_project_volunteer(request, application_id):
     volunteer_relation = VolunteerRelation.objects.get(id=application_id)
+    project = volunteer_relation.project
```

Comment:
I'd move this code to a helper method to specifically update the project timestamp.  It's not clear from the context why exactly we're saving the project all over the place.

Reply:
Moved to volunteer_relations.update_project_timestamp() helper method.